src/main/webapp/app/shared/util/tab.constants.ts

```diff
@@ -4,12 +4,12 @@ export const eventMainTabList: ITabInfo[] = [
   {
     tabName: 'Upcoming',
     tabTranslateKey: 'clubmanagementApp.event.upcoming',
-    tabRoute: '/event/upcoming'
+    tabRoute: '/entity/event/'
   },
   {
     tabName: 'Previous',
     tabTranslateKey: 'clubmanagementApp.event.previous',
-    tabRoute: '/event/previous'
+    tabRoute: '/entity/event?previous'
```

Comment:
the url is `?previous` or `/previous`

Reply:
The reason I used `?previous` here. If I `/previous` is used, when the admin delete or update an event, the url will become `entity/event/previous/:id/delete` or `entity/event/previous/:id/edit` respectively.

![image](https://user-images.githubusercontent.com/37231490/115731518-ca1b5c00-a3b9-11eb-843d-a93412682812.png)

Reply:
Oic
got it